packages/slice-machine/src/legacy/lib/models/common/widgets/ContentRelationship/index.ts

```diff
@@ -52,6 +52,7 @@ export const ContentRelationshipWidget: Widget<Link, typeof schema> = {
     config: {
       label,
       select: "document",
+      repeat: false,
```

Comment:
Still trying to understand why this is needed to make [this test](https://github.com/prismicio/slice-machine/blob/main/packages/slice-machine/test/lib/models/common/widgets/index.test.ts) pass

Reply:
which of those tests is failing? what does the error message say? can you reproduce it locally?

Reply:
![Screenshot 2024-11-04 at 10 18 46](https://github.com/user-attachments/assets/43fb0a3c-79f1-4304-8df4-3749adc8e0b4)

Reply:
I see now that it's because of `withFallback` [here](https://github.com/prismicio/prismic-types-internal/pull/89/files#diff-b7285001e1fb0c3aeffcc649ccae5c113c6c0755100b038b91e2c9937dd04231R82)